web/sites/default/config/easy_breadcrumb.settings.yml

```diff
@@ -4,7 +4,7 @@ applies_admin_routes: false
 include_invalid_paths: false
 excluded_paths: search
 replaced_titles: ''
-custom_paths: ''
+custom_paths: "/parks/recreation :: Parks and Recreation | /parks\r\n/parks/teenforce :: Parks and Recreation | /parks\r\nregex!/parks/([^/]*) ::  Parks and Recreation | /parks :: Find a park | /parks/search\r\n/procurement/sustainable-procurement-program :: Procurement | /business-opportunities\r\n/archives :: Auditor | /auditor\r\n/audit-services :: Auditor | /auditor\r\n/elections :: Auditor | /auditor\r\n/ombudsman :: Auditor | /auditor\r\n/lobbyist :: Auditor | /auditor\r\n/charter :: Charter, Code, and Policies | /charter-code-policies\r\n/code :: Charter, Code, and Policies | /charter-code-policies\r\n/policies :: Charter, Code, and Policies | /charter-code-policies\r\n/charter-code-policies/about-policies :: Charter, Code, and Policies | /charter-code-policies :: Portland Policy Documents | /policies\r\n/phb/nne-oversight ::  Housing Bureau | /phb :: North and Northeast Neighborhood Housing Strategy | /phb/nnehousing\r\n/accounting/aac :: Office of Management and Finance | /omf :: Revenue and Financial Services | /omf/brfs :: Accounting | /accounting\r\n/bps/planning/cic :: Planning and Sustainability | /bps :: Long-range Planning | /bps/planning :: Comprehensive Planning | /bps/planning/comp-plan\r\n/bps/climate-action/renewable-fuel-standard :: Planning and Sustainability | /bps\r\n/bps/planning/wpdx-town-center :: Planning and Sustainability | /bps\r\n/bps/master-recycling-program :: Planning and Sustainability | /bps :: Garbage, recycling, and composting | /bps/garbage-recycling\r\n/bps/public-trash-cans :: Planning and Sustainability | /bps :: Garbage, recycling, and composting | /bps/garbage-recycling\r\n/bps/waste-equity :: Planning and Sustainability | /bps :: Garbage, recycling, and composting | /bps/garbage-recycling\r\n/bps/waste-equity/waste-equity-group :: Planning and Sustainability | /bps :: Garbage, recycling, and composting | /bps/garbage-recycling :: Waste Equity | /bps/waste-equity\r\n/sustainabilityatwork :: Planning and Sustainability | /bps :: Garbage, recycling, and composting | /bps/garbage-recycling\r\n/last-thursdays ::  Transportation | /transportation\r\n/sunday-parkways ::  Transportation | /transportation"
```

Comment:
I think the "/accounting/aac: Office of Management and Finance" line can be removed since OMF and BRFS are now unpublished/defunct.

Just a comment: I'm a little worried about having to maintain this list now that it's in config instead of the menu/database. But not sure that anyone was maintaining the menu anyway 😆

Reply:
/accounting/aac removed. I think nobody remembered we still have this "Site menu". :)